guides/managing.md

```diff
@@ -2,125 +2,164 @@
 
 This is the content strategy for dxw Playbook.
 
```

Comment:
This file probably needs the `---` structure at the top as well. None of the linters or tests are catching it, so this is just a guess.